api/getjobs.go

```diff
@@ -0,0 +1,73 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/ONSdigital/go-ns/audit"
+	"github.com/ONSdigital/go-ns/common"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/pkg/errors"
+)
+
+func (api *ImportAPI) getJobsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	auditParams := common.Params{}
+	logData := log.Data{}
+
+	filtersQuery := r.URL.Query().Get("state")
+	var filterList []string
+	if filtersQuery == "" {
+		filterList = nil
```

Comment:
filterList would already be nil if you haven't already assigned to it

Reply:
Inherited code, but good spot I will update this.